.gitlab/source_test/tooling_unit_tests.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Unit test of internal python code
+invoke_unit_tests:
+  stage: source_test
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  needs: []
+  rules:
+    - !reference [.on_invoke_tasks_changes]
+  script:
+    - source /root/.bashrc
+    - python3 -m pip install -r tasks/libs/requirements-github.txt
+    - inv -e invoke-unit-tests
+
+kitchen_invoke_unit_tests:
+  stage: source_test
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
```

Comment:
Would deb image work for windows kitchen tests?

Reply:
In this case yes as it's testing the invoke tasks of kitchen (so the python code only). I assume there is few risk of misalignment of this between the python version we would have on the images (that should be the same by the way)